e-shop-website-with-ilb-ase/azuredeploy.json

```diff
@@ -0,0 +1,888 @@
+    "agSubnetName": "app-gateway-subnet",
+    "subnetRouteTableName": "[concat(parameters('existingAseName'), '-route-table')]",
+    "subnetNSGName": "[concat(parameters('existingAseName'), '-nsg')]",
+    "aspName": "[concat(parameters('existingAseName'), '-asp')]",
+    "webAppName": "[concat(parameters('existingAseName'), '-web')]",
+    "contentAppName": "[concat(parameters('existingAseName'), '-content')]",
+    "apiAppName": "[concat(parameters('existingAseName'), '-api')]",
+    "adminAppName": "[concat(parameters('existingAseName'), '-admin')]",
+    "appInsightsName": "[concat(parameters('existingAseName'), '-app-insights')]",
+    "sqlServerName": "[concat(parameters('existingAseName'), '-sql-server')]",
+    "storageAccountName": "[concat(replace(parameters('existingAseName'), '-', ''), '')]",
+    "redisCacheName": "[concat(parameters('existingAseName'), '-cache')]",
+    "redisCacheSubnetName": "redis-cache-subnet",
+    "appGatewayName": "[concat(parameters('existingAseName'), '-waf')]",
+    "appGatewayID": "[resourceId('Microsoft.Network/applicationGateways', variables('appGatewayName'))]",
+    "appGatewayPublicIPName": "[concat(parameters('existingAseName'), '-waf-ip')]",
+    "appGatewayPublicIPDnsPrefix": "[concat(parameters('existingAseName'), '-waf')]",
+    "cdnName": "[concat(parameters('existingAseName'), '-cdn')]",
+    "cdnWebAppEndpointName": "[concat(parameters('existingAseName'), '-cdn-web')]",
+    "cdnStorageEndpointName": "[concat(parameters('existingAseName'), '-cdn-storage')]",
+    "webAppExternalHostName": "[concat(variables('webAppName'), '.', parameters('dnsSuffix'))]",
+    "webAppInternalHostName": "[concat(variables('webAppName'), '.', parameters('existingASEDNSSuffix'))]",
+    "apiAppInternalHostName": "[concat(variables('apiAppName'), '.', parameters('existingASEDNSSuffix'))]",
+    "contentAppExternalHostName": "[concat(variables('contentAppName'), '.', parameters('dnsSuffix'))]",
+    "contentAppInternalHostName": "[concat(variables('contentAppName'), '.', parameters('existingASEDNSSuffix'))]",
+    "adminAppExternalHostName": "[concat(variables('adminAppName'), '.', parameters('dnsSuffix'))]",
+    "adminAppInternalHostName": "[concat(variables('adminAppName'), '.', parameters('existingASEDNSSuffix'))]",
+    "sourceCodeRepositoryURL": "https://github.com/OGCanviz/e-shop-website-with-ilb-ase"
+  },
+  "resources": [
+    {
+      "apiVersion": "2018-01-01",
+      "type": "Microsoft.Network/virtualNetworks/subnets",
+      "name": "[concat(variables('vnetName'), '/', variables('redisCacheSubnetName'))]",
+      "properties": {
+        "addressPrefix": "192.168.251.0/26"
+      }
+    },
+    {
+      "apiVersion": "2018-01-01",
+      "type": "Microsoft.Network/virtualNetworks/subnets",
+      "name": "[concat(variables('vnetName'), '/', variables('agSubnetName'))]",
+      "properties": {
+        "addressPrefix": "192.168.251.64/26"
+      },
+      "dependsOn": [
```

Comment:
move all the dependsOn above the properties object

Reply:
done